README.md

```diff
@@ -30,13 +30,13 @@ Clone this shop repository into the custom plugin folder. In your project direct
 ```
 
 #### Step 3
-Copy private asset files into your root application's /private folder
+Create /private/plugins/reaction-swag-shop/ folder in the root application and copy private asset files into it.
 ```
 $ cp -Rv <your-new-shop-directory-name>/imports/plugins/custom/reaction-swag-shop/private/* <your-new-shop-directory-name>/private/plugins/reaction-swag-shop/
 ```
 
 #### Step 4
-Copy public files into your root application's /public folder
+Create /public/plugins/reaction-swag-shop/ folder in the root application and copy public resource files into it.
```

Comment:
the problem I had here is that images need to be in an `images` directory but that's not made clear

Reply:
Not sure what you mean. The public image files (not product images) don't need to live in an /image subfolder. Am I missing something?

Reply:
https://github.com/reactioncommerce/reaction-swag-shop/blob/master/server/methods.js#L127

Reply:
The product images have to be in an images directory, but there is no images directory to copy,  that's what I am talking about

Reply:
@zenweasel This is for the private assets. Asking, because I've been thinking it's about Step 4 (Public resources).

Just to be sure: The `cp` command itself works correctly, right? So your suggestion basically is to tell the user, that the source directory needs to be copied recursively into the target directory?

Reply:
Not sure why we should mention the name of the subfolder `images` explicitly. In future there may also pop up other directory names that would have to be copied.

Reply:
Maybe we should just assume that the name is going to be `reaction-swag-shop` and just make this a script that does all this?

Reply:
@zenweasel The "asset provisioning" PR could be the script you're referring to. It does execute under the hood the `cp` commands above.

Reply:
> we should just assume that the name is going to be reaction-swag-shop

Yes, this is the way the asset provisioning PR does work for now.